packages/connectivity/src/index.ts

```diff
@@ -52,10 +52,8 @@ export {
   DestinationJson,
   DestinationsByType,
   DestinationForServiceBindingOptions,
```

Comment:
Unfortunately this type is still wrongly named. Seems we never deprecated it, but I guess it would be fine to fix the name here as well?

Reply:
Would you prefer `DestinationFromServiceBindingOptions`?